superset-frontend/src/views/CRUD/data/database/DatabaseList.tsx

```diff
@@ -96,6 +270,18 @@ function DatabaseList({ addDangerToast, addSuccessToast }: DatabaseListProps) {
           /* TODO: add database logic here */
         }}
       />
+
+      <ListView<DatabaseObject>
```

Comment:
question:  just wondering what does `<DatabaseObject>` do here

Reply:
It allows specifying the generic for the ListView props. This means that props like `renderCard` are not `(data: any) => ReactNode` and are instead `(data: DatabaseObject) => ReactNode`. 

You can read more about it here: https://mariusschulz.com/articles/passing-generics-to-jsx-elements-in-typescript